lib/handlers/quote/config/quote-speed-config.ts

```diff
@@ -0,0 +1,35 @@
+import { MapWithLowerCaseKey, ProtocolPoolSelection } from '@uniswap/smart-order-router'
+
+export type QuoteSpeedConfig = {
+  v2PoolSelection?: ProtocolPoolSelection;
+  v3PoolSelection?: ProtocolPoolSelection;
+  maxSwapsPerPath?: number;
+  maxSplits?: number;
+  distributionPercent?: number;
+}
+
+export const QUOTE_SPEED_MAP: { [key: string]: QuoteSpeedConfig } = {
+  'standard': {},
+  'fast': {
+    v2PoolSelection: {
+      topN: 1,
+      topNDirectSwaps: 1,
+      topNTokenInOut: 2,
+      topNSecondHop: 0,
+      topNWithEachBaseToken: 2,
+      topNWithBaseToken: 2,
+    },
+    v3PoolSelection: {
+      topN: 1,
+      topNDirectSwaps: 1,
+      topNTokenInOut: 2,
+      topNSecondHop: 0,
+      topNWithEachBaseToken: 2,
+      topNWithBaseToken: 2,
+    },
+    maxSwapsPerPath: 1,
+    maxSplits: 2,
+    distributionPercent: 10
```

Comment:
If we need to I think it would be fine to raise this to 20

Reply:
@willpote What's your opinion on forcing direct swap for the popular pairs, like ETH/USDC? Do we need to raise the gas limit in sor?